### PR TITLE
[ADD] P2022-312 Add gitWorkflow.md file

### PR DESCRIPTION
**Pull request by WDaniszewska**
Added file containing description of Git workflow.

### CODE REVIEW

File: gitWorkflow.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+### The aim of this document is to provide simple guidelines about Git workflow that is used in the Test Automation project.
+**Branches**  
+When someone start working on task he should create feature branch with name of the task he is currently working on PROPER PATH:
+
+feat/\{task number\}/\{task name\}
```
Comment (MarekPawlowskiQA) [inline gitWorkflow.md:5]: Please remove "/"
 
Example should look like feat/{task number}/{task name}

File: gitWorkflow.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+When PR is changing anything about framework itself, every reviewer should run it and check if this particular change work (MAC vs Linux vs Windows).  
+When there is many changes requested in PR after making changes review process should start from the beginning.  
+There shouldn’t be situation when PR is hanging **more than three days**.
+
+**Pull Request Merging**  
+When PR is created it can be merged to main only if this PR got two approves by team QA and one approve from leader QA and all of the merge conflicts and discussions are resolved.  
+All Browser Stack tests should be green on dedicated runners.  
+Creator of Pull Request should use “squash commit” option, the merge commit should look like:
+
+[ADD] \{Task number\} + short informational description
```
Comment (MarekPawlowskiQA) [inline gitWorkflow.md:23]: Also here: [ADD]{Task number} + short informational description

**Review by MarekPawlowskiQA — CHANGES_REQUESTED** [review]
Requested very minor changes

File: gitWorkflow.md (hunk 1 above)
Comment (WDaniszewska) [reply gitWorkflow.md]: Done

File: gitWorkflow.md (hunk 2 above)
Comment (WDaniszewska) [reply gitWorkflow.md]: Done

**Review by WDaniszewska — COMMENTED** [review]
Done

**Review by ospatrycja — APPROVED** [review]

**Review by KrzysztofGajQA — APPROVED** [review]
not LGTM at the moment

**Review by KrzysztofGajQA — CHANGES_REQUESTED** [review]
Hi, there is a typo in line 28, in word "resolved". The correct word should be "resolve".

**Comment by KrzysztofGajQA** [comment]
Hi, the word from line 28 should be in line 29 and vice-versa ;)
feat_P2022-313_correction

**Comment by WDaniszewska** [comment]
> Hi, the word from line 28 should be in line 29 and vice-versa ;) feat_P2022-313_correction

**Review by KrzysztofGajQA — DISMISSED** [review]
All green to me  👍  Good to squash!

**Review by MarekPawlowskiQA — DISMISSED** [review]
lgtm

File: gitWorkflow.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+feat/{task number}/{task name}
+
+All of the changes should be committed to this branch.  
+Commits should be informational.  
+After work is finished user should create Pull Request (PR) to main branch.  
+It is advised to merge main to feature branch to resolve all conflicts, before creating PR.  
+
+**Review**  
+When PR is created everyone should find time to check it (whenever its possible).
```
Comment (annaslodkowska) [inline gitWorkflow.md:13]: Should be "it's" or "it is"

Comment (WDaniszewska) [reply gitWorkflow.md]: Done

**Review by MarekPawlowskiQA — APPROVED** [review]
still lgtm. btw I know dismissing approve is annoying after each change, but later you will get used to it. If this will start to be a real problem I will remove this option  :)

**Review by ospatrycja — APPROVED** [review]

**Review by KrzysztofGajQA — APPROVED** [review]
LGTM :)

**Pull request merged**